components/common/NavbarSearch.js

```diff
@@ -1,4 +1,5 @@
-import { useState, useRef, useEffect } from 'react';
+'use client';
```

Comment:
We don't need to add `'use client'` to all components, only ones that are rendered directly by RSCs

Reply:
Ok. I added this when I was debugging and thought it resolved my problem, but it was probably something else :)